css/style.css

```diff
@@ -543,5 +1062,243 @@
   .arrow-downd:hover {
     transform: scale(1.2);
   }
-}
 
+  .future-drive {
+    display: flex;
+    min-height: 900px;
+    background-image: url(/img/2013-tesla-model-s-cockpit.png);
+    background-repeat: no-repeat;
+    background-position-y: 56px;
+  }
+
+  .future-drive__title {
+    margin: 0;
+    padding-top: 60px;
+    padding-left: 80px;
+    font-size: 70px;
+    line-height: 85px;
+    color: #e51937;
+  }
+
+  .future {
+    margin-left: auto;
+    width: 440px;
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .container-future {
+    margin: 0 auto;
+    max-width: 330px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6132629)